docs/search_syntax.md

```diff
@@ -0,0 +1,294 @@
+| ```run.created_at```   | Run creation datetime                               | ```numeric```    |
+| ```run.finalized_at``` | Run end datetime                                    | ```numeric```    |
+| ```run.metrics```      | Set of run metrics                                  | ```dictionary``` |
+
+## Search Metrics
+You can filter the metrics using the following metric attributes associated with the ```metric``` object:
+| Property                | Type          |
+| ----------------------- | ------------- |
+| ```metric.name```       | ```string```  |
+| ```metric.last```       | ```numeric``` |
+| ```metric.last_step```  | ```numeric``` |
+| ```metric.first_step``` | ```numeric``` |
+
+## Operations
+
+### String operations
+For the ```string``` attributes you can use the following comparison operator:
+- ``` == ```
+- ``` != ```
+- ``` in ```
+- ``` .startswith() ```
+- ``` .endswith() ```
+
+### Numeric operations
+For the ```numeric``` attributes you can use the following comparison operator:
+- ``` == ```
+- ``` != ```
+- ``` > ```
+- ``` >= ```
+- ``` < ```
+- ``` <= ```
+
+### Boolean operations
+For the ```boolean``` attributes you can use the following comparison operator:
+- ``` == ```
+- ``` != ```
+
+#### Implicit Boolean comparison
+You can also use boolean attributes directly in a conditional statement without any comparison operator. 
+
+Equivalent to comparing the attribute to True.
+
+```python
+run.archived  # Checks if run.archived is equal to True
+```
+Equivalent to comparing the attribute to False.
+```python
+not run.archived  # Checks if run.archived is equal to False
+```
+### Logical operations
+You can create complex search queries combining multiple conditions with logical operators.
+- ``` and ```
+- ``` or ```
+- ``` not ```
+
+
+
+## Search run examples
+
+### Example with ```run.name``` (string)
+Select only the runs where the name exactly matches "TestRun1"
+
+```python
+run.name == "TestRun1"
+```
+
+Select only the runs where the name is different from "TestRun1"
+```python
+run.name != "TestRun1"
+```
+
+Select only the runs where "Run1" is contained in the run name
+```python
+"Run1" in run.name
+```
+
+Select only the runs where the name starts with "Test"
+```python
+run.name.startswith('Test')
+```
+
+Select only the runs where the name ends with "Run1"
+```python
+run.name.endswith('Run1')
+```
+
+### Example with ```run.duration``` (numeric)
+
+Select only the runs where the duration is exactly 60
+
+```python
+run.duration == 60
+```
+
+
+Select only the runs where the duration is not 60
+```python
+run.duration != 60
+```
+
+Select only the runs where the duration is greater than 60
+```python
+run.duration > 60
+```
+
+Select only the runs where the duration is greater or equal to 60
+```python
+run.duration >= 60
+```
+
+Select only the runs where the duration is less than 60
+```python
+run.duration < 60
+```
```

Comment:
we shouldn't list _all_ the numerical comparison operators here — let's just give 2 or 3 examples, like `==`, `>=`, and `<`

Reply:
same remark for the `metric.last` example